### PR TITLE
Fixes for new lands stuff (#4)

### PR DESCRIPTION
**Pull request by Wirednuke**

### CODE REVIEW

**Comment by Chilly021** [comment]
Huge thanks to @gnummers  on the follow up!

**Review by Chilly021 — DISMISSED** [review]

**Review by Leolus — DISMISSED** [review]

File: Segments/Bloodlands.mapproj [hunk 1]
```diff
@@ -774,14 +774,14 @@
         /// <summary>
         /// Initializes a new instance of the <see cref="CrystalGauntlets"/> class.
         /// </summary>
-        public BloodGuantlets() : base(71)
+        public BloodGuantlets() : base(37)
```
Comment (sproatba) [inline Segments/Bloodlands.mapproj:777]: Gauntlet typo

**Review by sproatba — CHANGES_REQUESTED** [review]

File: Segments/Bloodlands.mapproj (hunk 1 above)
Comment (Chilly021) [reply Segments/Bloodlands.mapproj:777]: Good catch!

Comment (Chilly021) [reply Segments/Bloodlands.mapproj:777]: Resolved.

**Review by Chilly021 — APPROVED** [review]

**Review by Leolus — APPROVED** [review]

**Review by sproatba — APPROVED** [review]

**Pull request merged**